editor/progress_dialog.cpp

```diff
@@ -126,6 +127,22 @@ void BackgroundProgress::end_task(const String &p_task) {
 
 ProgressDialog *ProgressDialog::singleton = nullptr;
 
+void ProgressDialog::_notification(int p_what) {
+	switch (p_what) {
+		case NOTIFICATION_ENTER_TREE:
```

Comment:
```suggestion
```
`NOTIFICATION_THEME_CHANGED` alone is enough.

Reply:
Done!